Solutions/Lookout/Data Connectors/Logo/lookout.svg

```diff
@@ -1,11 +1,3 @@
-<?xml version="1.0" standalone="no"?>
-<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
- "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
-<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
- width="72.000000pt" height="72.000000pt" viewBox="0 0 72.000000 72.000000"
- preserveAspectRatio="xMidYMid meet">
-
-<g transform="translate(0.000000,72.000000) scale(0.100000,-0.100000)"
-fill="#000000" stroke="none">
-</g>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="46.352" height="45.294" viewBox="0 0 46.352 45.294">

```

Comment:
@rajendrakhabiyagate6 , svg code file have invalid attributes. please follow the below guidelines:

Logo needs to be in SVG format and under 5 Kb
Ensure raw file of logo does not have any of the following:
a) cls and style formats
b) embedded png formats
c) xmlns:xlink
**d)data-name**
Do not use xlink:href - use inline instead
Do not use title tag
**If some properties in the logo have IDs (for e.g. <g id="layer0"...), then set these IDs as GUIDs so that these are uniquely identifiable across all Azure logo assets**
**Logo scales well to fit in a 75 px square**
SVG code file is formatted well for readability
for more detail please follow the link:
[https://github.com/Azure/Azure-Sentinel/tree/master/DataConnectors#build-the-connector](https://github.com/Azure/Azure-Sentinel/tree/master/DataConnectors#build-the-connector)

Reply:
@v-maudan SVG file Updated